libmscore/layout.cpp

```diff
@@ -2238,7 +2238,8 @@ void Score::createBeams(Measure* measure)
                                     if (!beamNoContinue(prevCR->beamMode())
                                         && !pm->lineBreak() && !pm->pageBreak() && !pm->sectionBreak()) {
                                           beam = prevCR->beam();
-                                          a1 = beam ? beam->elements().front() : prevCR;
+                                          //a1 = beam ? beam->elements().front() : prevCR;
+										  a1 = beam ? nullptr : prevCR; // when beam is found, a1 is no longer required.
```

Comment:
Something is wrong with tabs/spaces here and below. We prefer 6 spaces. Could you fix please?

Reply:
OK, sorry for this dumb mistake...